mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -17,6 +18,10 @@ export const UserPage = () => {
         level={HeadingLevel.h2}
         text="User page"
       />
+      <Button
+        value="Create new way"
+        onClick={() => WayPreviewDAL.createWayPreview(uuid!)}
```

Comment:
Can you get rid of using the exclamation mark?

Reply:
No I don't know how

Reply:
Added interface for property from useParams but dont know is it right way